presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -183,6 +183,7 @@
     private boolean optimizedRepartitioningEnabled;
 
     private boolean pushdownSubfieldsEnabled;
+    private boolean pushdownSubfieldsFromLambdaEnabled;
```

Comment:
I am wondering if we need a separate config for this.

Reply:
I think it could be useful during the rollout phase.